apps/testing-javascript/package.json

```diff
@@ -2,7 +2,8 @@
   "name": "testing-javascript",
   "private": true,
   "scripts": {
-    "dev": "next dev -p 3018",
+    "dev": "next dev -p 3020",
+    "dev:sanity": "sanity dev",
```

Comment:
run sanity studio locally, this is a great way to start up the studio so that you can browse the production data that has been seeded at `localhost:3333`